supplement.csv

```diff
@@ -1,5 +1,18 @@
 Lemma,Sense,Word Class,Year,Quotation,Source,Note
+愛國者,/,noun,1901,?,梁啓超『梁啓超商會議、國家論、飲冰室自由書 』,
+愛情,/,noun,1584,必生事物之象面含存之司愛欲者愛一物、必生一愛情面内含存之,羅明堅、陽麗諾『天主聖教實錄』,
+愛人,2,noun,1908,"one in love, 愛人, 相好, 情人","顏惠慶『英華大辭典』s.v. Lover+",
+安樂椅,/,noun,1908,"A soft arm-chair, 偃息椅, 安樂椅, 軟熟圈手椅, 休息椅","顏惠慶『英華大辭典』s.v. Easy chair",
+安樂椅子,/,noun,1913,easy chair,"A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",
 暗生植物,/,noun,1887,Cryptogam{e}æ 暗生植物,Fryer『西藥大成藥品中西名目表』18,dog-Latin: Kretschmer §31
+暗室,/,noun,1664,故山市海市、暗室一隙、皆得転吸面見之、大鏡之中,方以智『物理小識』,
+罷工,2,verb,1865,"罷工 [...] in the ordinary sense, to strike work for a rise of wages, as is often done by the Canton weavers.",R. Morrison『五車韻府』,
```

Comment:
```suggestion
罷工,2,verb,1865,"罷工 ... in the ordinary sense, to strike work for a rise of wages, as is often done by the Canton weavers",R. Morrison『五車韻府』,
```
1. Square brackets not needed
2. No complete main clause here, so period not needed